clients/src/main/resources/common/message/VoteRequest.json

```diff
@@ -21,7 +21,7 @@
   "validVersions": "0",
   "flexibleVersions": "0+",
   "fields": [
-    { "name": "ClusterId", "type": "string", "versions": "0+",
+    { "name": "ClusterId", "type": "string", "versions": "0+",  "ignorable": true,
```

Comment:
Why do we need this to be ignorable?

Reply:
this was checked in by mistake when testing, I will revert it.